lib/relay.js

```diff
@@ -43,6 +72,29 @@ function requestHandler(filterRules) {
 
       req.url = result.url;
       logContext.ioUrl = result.url;
+
+      // check if this is a streaming request for binary data
+      if (req.headers && binaryRequest.indexOf(req.headers['accept']) !== -1) {
```

Comment:
Identify the streaming request by the accept header.

Reply:
consider `req.get('accept')` instead?

Reply:
Is `req.headers &&` necessary?